README.md

```diff
@@ -61,3 +75,173 @@ psql -U postgres -d registry -h host.docker.internal -f /opt/src/db/sql/0001.sql
 Seeing an unresolved image or not deploying the API? Remember to tag:
 oc tag platsrv-registry-api:latest platsrv-registry-api:dev
 
+ult -n devops-security-aporeto-operator
```

Comment:
what are the logs for?

Reply:
Those are just notes in the README. I'm currently working through better documentation.